src/lib/guesstimator/formatter/formatters/DistributionTextProportion.js

```diff
@@ -1,31 +1,9 @@
-import {parseNumber} from './lib.js'
+import {regexBasedFormatter} from './lib.js'
+
+const PROPORTION_REGEX = /^\s*(\d*(?:\.\d+)?)\s?(M|B|K|T)?\s*(?:of|in)\s*(\d*(?:\.\d+)?)\s?(M|B|K|T)?\s*$/
```

Comment:
Any chance this could be broken into pieces or something?  These are quite long, hard to interpret